tests/jenkins/jobs/BuildManifest_Jenkinsfile

```diff
@@ -22,8 +22,10 @@ pipeline {
                     echo buildManifest.build.getFilename()
                     echo buildManifest.getArtifactRoot('bundle-build', '1')
                     echo buildManifest.getArtifactRootUrl('https://ci.opensearch.org/ci/dbc', 'bundle-build', '1')
+                    echo buildManifest.build.getPackageName()
+                    echo buildManifest.components.get("OpenSearch").artifacts.get("dist").first()
```

Comment:
This should be extracted into a method, I want to be able to write:

```
buildManifest.components.getMinArtifact()
```

The implementation of `getMinArtifact` should know which component to get (the first one? the one that matches the name of the build artifact?) and raise errors if such a component cannot be found, or artifacts is missing, or the artifact named "dist" is missing, so it's easier to debug.